src/emulator/emulator.cpp

```diff
@@ -28,12 +28,8 @@ extern "C"
  */
 
 // Note thread-locality here
-static thread_local faabric::Message _emulatedCall = faabric::Message();
```

Comment:
The emulator is almost useless now. It existed before Faabric and was used to execute native benchmarks with Faasm. Things have moved on a lot, so now it's just used for low-level state testing (which can eventually be moved to Faabric as well)